### ISSUE TITLE
IndexOutOfBoundsException

### ISSUE THREAD

**Issue by msya** [linkedin/Spyglass#11]
An IndexOutofBoundsException occurs if you follow these steps: 

1. Write some text and do a mentions such as "Hello @Linkedin test".
2. Copy the text "in test" and paste it at the same place.
3. The following exception will occur: 

```
java.lang.IndexOutOfBoundsException: replace (28 ... 36) ends beyond length 32
      at android.text.SpannableStringBuilder.checkRange(SpannableStringBuilder.java:1022)
      at android.text.SpannableStringBuilder.replace(SpannableStringBuilder.java:462)
      at com.linkedin.android.spyglass.mentions.MentionsEditable.replace(MentionsEditable.java:113)
      at android.text.SpannableStringBuilder.replace(SpannableStringBuilder.java:456)
      at android.text.SpannableStringBuilder.replace(SpannableStringBuilder.java:33)
      at com.linkedin.android.spyglass.ui.MentionsEditText.ensureMentionSpanIntegrity(MentionsEditText.java:574)
      at com.linkedin.android.spyglass.ui.MentionsEditText.access$900(MentionsEditText.java:56)
      at com.linkedin.android.spyglass.ui.MentionsEditText$MyWatcher.afterTextChanged(MentionsEditText.java:381)
      at android.widget.TextView.sendAfterTextChanged(TextView.java:9007)
      at android.widget.TextView$ChangeWatcher.afterTextChanged(TextView.java:11588)
      at android.text.SpannableStringBuilder.sendAfterTextChanged(SpannableStringBuilder.java:976)
      at android.text.SpannableStringBuilder.replace(SpannableStringBuilder.java:520)
      at com.linkedin.android.spyglass.mentions.MentionsEditable.replace(MentionsEditable.java:113)
      at android.text.SpannableStringBuilder.replace(SpannableStringBuilder.java:456)
      at android.text.SpannableStringBuilder.replace(SpannableStringBuilder.java:33)
      at android.widget.TextView.paste(TextView.java:10898)
      at android.widget.TextView.onTextContextMenuItem(TextView.java:10316)
```

If I follow the same scenario in the LinkedIn app, this exception doesn't occur.

**Comment by msya** [comment]
@nhibner Would you please release the updated library to maven central?

**Comment by nhibner** [comment]
@msya Yes, I'll have a new release up today. It will also fix the accessibility issue (#11) in this release.
